### ISSUE TITLE
Allow name change for the printed bed

### ISSUE THREAD

**Issue by Cobnut3D** [bambulab/BambuStudio#3359]
I often print multiple models one after the other, leaving my slicer open all day.  WIth other slicers, the exported GCode (or file) is named after the first object placed on the bed, but Bambu Studio appears to use the first object loaded for the entire time the program is open, regardless of what's on the bed.

Request:  Allow the name of the printed bed to be changed without restarting the program.

**Comment by Sonicboom247** [comment]
i agree, this is quite annoying it does not take the name of the sliced file. Instead, it holds on to the name of the first object.  The only way i can get around this is to click add a plate, delete the old plate, and add the new object to the new plate.

**Comment by Haidiye01** [comment]
Can you take screenshots or record a screen to illustrate your needs

**Comment by Cobnut3D** [comment]
Thanks for the response.  The issue is with the naming of the model being
printed.  When you first open Bambu Studio the name is set to the name of
the first object printed and it remains that name even when that object is
deleted and other objects are loaded.  I've printed for an entire day and
by the late evening the prints are still named after an object I printed in
the morning.  There is no way to change this name, no way to set it
manually to make more sense.  Other slicers allow this.

This is really important when you're printing a lot of objects with similar
coded names and need to keep close track of what's being printed.

bstudio_1.png shows the "Prepare" tab.  Note that the name at the top is
"039_1d.stl" but the only object loaded is 039_1e.stl.

bstudio_2.png is the "Device" tab.  The name of the object being printed is
"039_1d", but it's actually 039_1e being printed.

bstudio_3.png is a screenshot from the Bambu App showing the
notifications.  Note that all the notifications have the same name, even
though they're clearly different objects.

Thanks.

Jon

**Comment by Cobnut3D** [comment]
[image: bambu_1.jpg][image: bambu_2.jpg][image: bstudio_3.png]

**Comment by Haidiye01** [comment]
Did your picture upload successfully？What I see is this，I didn't see anything
![image](https://github.com/bambulab/BambuStudio/assets/40328051/b274d565-68d1-46fd-a7cc-65c246684f02)

**Comment by Haidiye00** [comment]
Please download and use the latest version. If there are still any issues, please use pictures or screenshots to describe your ideas. Thank you
